docs/api/toast.md

```diff
@@ -86,6 +86,50 @@ import ThemingPlayground from '@site/static/usage/v7/toast/theming/index.md';
 
 <ThemingPlayground />
 
+## Accessibility
+
+### Focus Management
+
+Toasts are intended to be subtle notifications and are not intended to interrupt the user. User interaction should not be required to dismiss the toast. As a result, focus is not automatically moved to a toast when one is presented.
+
+### Screen Readers
+
+Toasts set aria properties in order to be [accessible](../reference/glossary#a11y) to screen readers, but these properties can be overridden if they aren't descriptive enough or don't align with how the toast is being used in an app.
+
+#### Role
+
+`ion-toast` has `role="status"` and `aria-live="polite"` set on the inner `.toast-content` element. This causes screen readers to only announce the toast message and header. Buttons and icons will not be announced.
+
+`aria-live` causes screen readers to announce the content of the toast when it is updated. However, since the attribute is set to `'polite'`, screen readers should not interrupt the current task.
+
+Since toasts are intended to be subtle notification, `aria-live` should never be set to `"assertive"`. If developers need to interrupt the user with an important message, we recommend using an [alert](./alert).
+
+#### Toast Buttons Description
+
+Buttons containing text will be read by a screen reader. If a button contains only an icon, or a description other than the existing text is desired, a label should be assigned to the button by passing `aria-label` to the `htmlAttributes` property on the button.
```

Comment:
> Buttons containing text will be read by a screen reader. 

Is this even accurate? The section above it says:

> `ion-toast` has `role="status"` and `aria-live="polite"` set on the inner `.toast-content` element. This causes screen readers to only announce the toast message and header. Buttons and icons will not be announced.

But the original issue this is for is for toast and states:

> Buttons in an ion-toast that are only an icon (eg close) cannot be made accessible as they do not have an aria-label property that can be set.

So I think I should update this to say that they won't be read unless `aria-label` is provided.

Reply:
I think what happens is that it doesn't announce the text or aria-label when the toast appears on the screen, but it will announce it if the user navigates to the toast. I suppose the current text is a bit confusing, but I think it's all correct.

Reply:
Done: https://github.com/ionic-team/ionic-docs/pull/3064/commits/4229be3de4f1bb6719f8779024c31ec42754cd4f